ts/features/profile/store/reducers/profile.ts

```diff
@@ -24,4 +28,47 @@ const profileReducer = (
   return state;
 };
 
+export const profileSelector = (state: GlobalState): ProfileState =>
+  state.profile;
```

Comment:
This is not your reducer.
```suggestion
  state.features.profile;
```